tests/test_read_CuratedCalibs.py

```diff
@@ -33,12 +33,14 @@
 import lsst.daf.persistence as dafPersist
 
 ROOT = os.path.join(getPackageDir('obs_base'), 'tests')
```

Comment:
To prevent the duplication of the test how about adding:

```python
if not os.path.exists(ROOT):
    ROOT = None
```
and then the `skipIf` is `@unittest.skipIf(ROOT is None, ROOTSkipMsg)`
?

My try version was being a bit more defensive around `obs_base` not being available but you are right that if `obs_base` is not setup then everything is going to break anyhow.